tests/integration/test_end_to_end_dcp.py

```diff
@@ -163,5 +163,73 @@ def test_10x_run(self):
             raise TimeoutError("test timed out")
 
 
+class TestOptimusRun(TestEndToEndDCP):
+
+    OPTIMUS_10X_ANALYSIS_OUTPUT_FILES_REGEXES = [
+        re.compile('merged\.bam$'),
+        re.compile('sparse\_counts\.npz$'),
+        re.compile('sparse\_counts\_row\_index\.npy$'),
+        re.compile('sparse\_counts\_col\_index\.npy$'),
+        re.compile('merged-cell-metrics\.csv\.gz$'),
+        re.compile('merged-gene-metrics\.csv\.gz$'),
+        re.compile('empty\_drops\_result\.csv$'),
+        re.compile('^.+zarr!\.zattrs$'),
+        re.compile('^.+zarr!\.zgroup$'),
+        re.compile('^.+zarr!expression_matrix!\.zgroup$'),
+        re.compile('^.+zarr!expression_matrix!cell_id!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!cell_id!0$'),
+        re.compile('^.+zarr!expression_matrix!cell_id!1$'),
+        re.compile('^.+zarr!expression_matrix!cell_metadata_numeric!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!cell_metadata_numeric!0\.0$'),
+        re.compile('^.+zarr!expression_matrix!cell_metadata_numeric_name!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!cell_metadata_numeric_name!0$'),
+        re.compile('^.+zarr!expression_matrix!expression!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!expression!0\.0$'),
+        re.compile('^.+zarr!expression_matrix!expression!0\.1$'),
+        re.compile('^.+zarr!expression_matrix!expression!0\.2$'),
+        re.compile('^.+zarr!expression_matrix!expression!0\.3$'),
+        re.compile('^.+zarr!expression_matrix!expression!0\.4$'),
+        re.compile('^.+zarr!expression_matrix!expression!0\.5$'),
+        re.compile('^.+zarr!expression_matrix!expression!1\.0$'),
+        re.compile('^.+zarr!expression_matrix!expression!1\.1$'),
+        re.compile('^.+zarr!expression_matrix!expression!1\.2$'),
+        re.compile('^.+zarr!expression_matrix!expression!1\.3$'),
+        re.compile('^.+zarr!expression_matrix!expression!1\.4$'),
+        re.compile('^.+zarr!expression_matrix!expression!1\.5$'),
+        re.compile('^.+zarr!expression_matrix!gene_id!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!gene_id!0$'),
+        re.compile('^.+zarr!expression_matrix!gene_id!1$'),
+        re.compile('^.+zarr!expression_matrix!gene_id!2$'),
+        re.compile('^.+zarr!expression_matrix!gene_id!3$'),
+        re.compile('^.+zarr!expression_matrix!gene_id!4$'),
+        re.compile('^.+zarr!expression_matrix!gene_id!5$'),
+        re.compile('^.+zarr!expression_matrix!gene_metadata_numeric!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!gene_metadata_numeric!0\.0$'),
+        re.compile('^.+zarr!expression_matrix!gene_metadata_numeric_name!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!gene_metadata_numeric_name!0$')
+    ]
+
+    def test_optimus_run(self):
+        runner = DatasetRunner(deployment=self.deployment)
+
+        with Timeout(210 * 60) as to:  # timeout after 3 hours
```

Comment:
Are the units seconds here? Makes 3.5 hours if so

Reply:
Yes! I forgot to change the comment last time

Reply:
It'd be great if the comments can make it clear that this number is consistent with what we set in the Gitlab runner settings.